<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GanttWidget</class>
 <widget class="QWidget" name="GanttWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GanttWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="GanttPlayerSettings" name="playerSettings" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="GanttPlayerControl" name="playerControl" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="GanttIntervalSlider" name="widgetIntervalSlider" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GanttDtLine" name="widgetDtLine" native="true"/>
     </item>
     <item>
      <widget class="GanttGraphicsView" name="ganttView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GanttPlayerControl</class>
   <extends>QWidget</extends>
   <header>ganttplayercontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GanttPlayerSettings</class>
   <extends>QWidget</extends>
   <header>ganttplayersettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GanttGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ganttgraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>GanttIntervalSlider</class>
   <extends>QWidget</extends>
   <header>ganttintervalslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GanttDtLine</class>
   <extends>QWidget</extends>
   <header>ganttdtline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
